test/components/error_spec.js

```diff
@@ -0,0 +1,40 @@
+import PlayerError from '../../src/components/error'
+import Core from '../../src/components/core'
+import Events from '../../src/base/events'
+
+describe('PlayerError', function() {
+  beforeEach(function() {
+    this.core = new Core({})
+
+    this.errorData = {
+      code: 'test_01',
+      description: 'test error',
+      level: PlayerError.Levels.FATAL,
+      origin: 'test',
+      scope: 'it',
+      raw: {},
+    }
+  })
+
+  it('trigger ERROR with error data when error method is called', function() {
```

Comment:
`it triggers the` or we could:

```javascript
describe("when error method is called", function(){
    it("triggers the event ERROR", ...
})
``` 

The fact of ERRO being fired with error data is already expected.

Reply:
fixed on e6917a0eeb6bacfe6985cefc5a147bbcc12681cb and 8404cf9a80e2273e3879e426597d5f70a1960294